ansible-scylla-node/tasks/common.yml

```diff
@@ -277,3 +277,20 @@
   become: true
   when: update_sysctl|bool is defined and update_sysctl|bool
 
+- name: Deactivate firewall
```

Comment:
I'd still be happy if you could test the whole deploy and ideally even with a 3 node cluster (I will try to merge the dev branch with molecule tests asap).
Why?
Because the task that checks if cluster is up is far far before this task and it would fail, since the firewall is up back then.
That said I think this task should be properly split to both RHEL and Debian yml files and do proper stuff there without ignore-ing the errors.
The this file will work as charm, since cluster will be created.